demos/vc_issuer/src/main.rs

```diff
@@ -172,14 +173,15 @@ fn authorize_vc_request(
         let config = config.get();
 
         for idp_canister_id in &config.idp_canister_ids {
-            if let Ok(alias_tuple) = get_verified_id_alias_from_jws(
+            match get_verified_id_alias_from_jws(
```

Comment:
Why was this logic changed? Is it somehow related to the derivation origin?

Reply:
The logic remains the same, the change is just to print the error message in case of verification error (previously the error message was ignored).

Reply:
@frederikrothenberger sorry this is my code, the original PR description including this:

> Various error logging improvements in the issuer & the vc flow


see here: https://github.com/dfinity/internet-identity/pull/2335

I don't feel too strongly about it